launch_yaml/test/launch_yaml/test_executable.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test parsing an executable action."""
+
+from pathlib import Path
+
+from launch import LaunchService
+
+from launch_frontend import Parser
+
+
+def test_executable():
+    """Parse executable yaml example."""
+    root_entity, parser = Parser.load(str(Path(__file__).parent / 'executable.yaml'))
```

Comment:
@ivanpauno same comment about parametrizing our tests by the launch description markup as in https://github.com/ros2/launch_ros/pull/23#discussion_r284335310.

Reply:
But, where should I put them? Is ok putting them in launch_frontend? And having launch_xml and launch_yaml as test dependencies.
Or maybe, a new package only for the tests.

Reply:
Hmm, good point. Ok, nevermind my comment, it makes it more obscure than what it's worth.